client/executor/src/integration_tests/mod.rs

```diff
@@ -523,7 +523,7 @@ fn offchain_http_should_work(wasm_method: WasmExecutionMethod) {
 
 #[test_case(WasmExecutionMethod::Interpreted)]
 #[cfg_attr(feature = "wasmtime", test_case(WasmExecutionMethod::Compiled))]
-#[should_panic(expected = "Allocator ran out of space")]
+#[should_panic]
```

Comment:
Nope, not a good idea to do this. This means any exception will be accepted. Please revert

Reply:
By default the debug message is used, which is now the full chain of errors which is not something we want to match against. I'll see how the functionality can be restored.

Reply:
@bkchr would it be acceptable that the two error messages differ slightly - native could have a chain, where `wasmi` could not? Or must they be *_exact_* matches?

Reply:
Can we not just match on parts of the error message?

Reply:
So remove the should_panic and do `unwrap_error().to_string().contains()`?

Reply:
> So remove the should_panic and do `unwrap_error().to_string().contains()`?

That implies a yes, the error types do not have to be the exact equiv :)

> Can we not just match on parts of the error message?

Technically if only the first error is formatted, equiv relation should be sufficient..